projects/emissary-ingress/emissary/Makefile

```diff
@@ -28,14 +28,30 @@ AMBASSADOR_VERSION_TARGET=$(REPO)/python/ambassador.version
 
 include $(BASE_DIRECTORY)/Common.mk
 
-
 $(GO_MOD_DOWNLOAD_TARGETS): $(AMBASSADOR_VERSION_TARGET)
 
 # do not pass our makeflags down to emissary make since there will be unncessary warnings
 $(AMBASSADOR_VERSION_TARGET): MAKEFLAGS=
 $(AMBASSADOR_VERSION_TARGET):
-	@export VERSION="v3.0.0" && export CHART_VERSION=$(CHART_VERSION) && cd emissary && GOPATH=/var && make python/ambassador.version
-
+	@export VERSION="v3.3.0" && export CHART_VERSION=$(CHART_VERSION) && cd emissary && GOPATH=/var && make python/ambassador.version
+
```

Comment:
We don't have a better strategy than this? Any other repos building multiple charts now?

Reply:
This one is specific to make a certain file `python/ambassador.version` required by emissary during the build process. I could try to patch it out, but it would require changing some python code which I wanted to avoid.